### DISCUSSION TITLE
The new Lunar looks kinda familiar... ;)

### DISCUSSION THREAD

**Discussion by waydabber** [MonitorControl/MonitorControl#768] [General]
I think the new [Lunar 5](https://lunar.fyi/changelog#5.0.0) was inspired by the best Brightness Control app in the Universe *:

<img width="382" alt="Screen Shot 2021-11-04 at 15 18 52" src="https://user-images.githubusercontent.com/37590873/140330962-5dc0743d-a78f-40e0-88b4-3a1ee76414ed.png">

_(Just to clarify: Lunar is a great app, @alin23 specifically asked for permission beforehand to use my sliders and I helped out testing stuff as well + @alin23 really helped out with MonitorControl from time to time, we are buddies!)_

_* Mr. Obvious would point out that this line refers to MonitorControl of course :)._

**Comment by alin23** [comment]
What a rip off!!

**Reply by waydabber** [reply]
lol... I wonder who could do such an outrageous thing!
